### PR TITLE
Migrate GitHub Issues to 1.0.0-preview

### PR DESCRIPTION
**Pull request by prathyusha12345**
Migrated the project to Ml.Net 1.0.0-preview

### CODE REVIEW

File: machine-learning/tutorials/GitHubIssueClassification/Program.cs [hunk 1]
```diff
@@ -135,7 +132,7 @@ public static IEstimator<ITransformer> BuildAndTrainModel(IDataView trainingData
 
         }
 
-        public static void Evaluate()
+        public static void Evaluate(DataViewSchema trainigDataViewSchema)
```
Comment (JRAlexander) [inline machine-learning/tutorials/GitHubIssueClassification/Program.cs]: trainigDataViewSchema is misspelled.

File: machine-learning/tutorials/GitHubIssueClassification/Program.cs [hunk 2]
```diff
@@ -155,28 +152,24 @@ public static void Evaluate()
...
+            Console.WriteLine($"*       MacroAccuracy:    {testMetrics.MacroAccuracy:0.###}");
             Console.WriteLine($"*       LogLoss:          {testMetrics.LogLoss:#.###}");
             Console.WriteLine($"*       LogLossReduction: {testMetrics.LogLossReduction:#.###}");
             Console.WriteLine($"*************************************************************************************************************");
             // </SnippetDisplayMetrics>
 
             // Save the new model to .ZIP file
             // <SnippetCallSaveModel>
-            SaveModelAsFile(_mlContext, _trainedModel);
+            SaveModelAsFile(_mlContext, trainigDataViewSchema, _trainedModel);
```
Comment (JRAlexander) [inline machine-learning/tutorials/GitHubIssueClassification/Program.cs]: trainigDataViewSchema is misspelled.

File: machine-learning/tutorials/GitHubIssueClassification/Program.cs [hunk 3]
```diff
@@ -198,11 +191,10 @@ public static void PredictIssue()
 
         }
 
-        private static void SaveModelAsFile(MLContext mlContext, ITransformer model)
+        private static void SaveModelAsFile(MLContext mlContext,DataViewSchema trainigDataViewSchema, ITransformer model)
```
Comment (JRAlexander) [inline machine-learning/tutorials/GitHubIssueClassification/Program.cs]: trainigDataViewSchema is misspelled

**Comment by prathyusha12345** [comment]
Thanks for identifying. I fixed the spelling. Can you check once

**Comment by JRAlexander** [comment]
Nice work, @prathyusha12345!

**Comment by JRAlexander** [comment]
@cjgronlund - Do you explicitly approve this PR for BUILD ask mode?

**Comment by cjgronlund** [comment]
Approve.

**Pull request merged**